jib-core/src/main/java/com/google/cloud/tools/jib/api/DockerDaemonImage.java

```diff
@@ -56,8 +62,11 @@ public static DockerDaemonImage named(String imageReference)
 
   private final ImageReference imageReference;
   private Path dockerExecutable = Paths.get("docker");
+  private Map<String, String> dockerEnvironment = Collections.emptyMap();
```

Comment:
As I mention in the `DockerClient` change, this is a change in behaviour as it will cause the environment to be replaced.

Reply:
Same as above. In my opinion, `dockerExecutable` and `dockerEnvironment` should be removed from this class and left in `DockerClient` in order to eliminate duplication.

Reply:
Yeah, I think it makes sense to remove both `setDockerExecutable` and `setDockerEnvironment` and instead have `setDockerClient()`.